fcli-core/fcli-ssc/src/main/java/com/fortify/cli/ssc/appversion/helper/SSCAppVersionCreateCopyFromBuilder.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright 2021, 2023 Open Text.
+ *
+ * The only warranties for products and services of Open Text 
+ * and its affiliates and licensors ("Open Text") are as may 
+ * be set forth in the express warranty statements accompanying 
+ * such products and services. Nothing herein should be construed 
+ * as constituting an additional warranty. Open Text shall not be 
+ * liable for technical or editorial errors or omissions contained 
+ * herein. The information contained herein is subject to change 
+ * without notice.
+ *******************************************************************************/
+package com.fortify.cli.ssc.appversion.helper;
+
+import com.fortify.cli.ssc._common.rest.SSCUrls;
+import com.fortify.cli.ssc.appversion.cli.mixin.SSCCopyOptions;
+import kong.unirest.HttpRequest;
+import kong.unirest.UnirestInstance;
+
+import java.util.HashMap;
+
+public final class SSCAppVersionCreateCopyFromBuilder {
```

Comment:
I'll review this later; based on earlier comments, I think there may be some changes in this class as well. Note though that classes in the `helper` package shouldn't depend on classes in `cli` packages as this creates circular package dependencies. So, the `SSCCopyOptions` enum should be moved to this class/package, or decoupled in some other way.